source/_redirects

```diff
@@ -472,3 +472,5 @@
 /integrations/yessssms /more-info/removed-integration 301
 /integrations/yweather /more-info/removed-integration 301
 /components/zwave /integrations/zwave_js
+/components/smarthab /more-info/removed-integration 301
```

Comment:
We have updated the our redirect system last week, so we no longer need this one :)

```suggestion 
```

Reply:
Great, thanks :)